### DISCUSSION TITLE
The given data was invalid.

### DISCUSSION THREAD

**Discussion by isaacdarcilla** [livewire/livewire#5060] [1. Help] [answered]
I have this 5MB image. When I try to upload it this occurs. (See screenshot)

![image](https://user-images.githubusercontent.com/22732118/175201443-b6775d4e-40a4-4232-a2c9-86c43697a658.png)

In my network tab in Chrome, 

I got this error in `livewire/upload-file?expires=1655954899&signature=f45d3d7534279d2d1f0d9207be25c03730cda4c0fa799cd955879430fbd58700` that says 'The given data was invalid. (See screenhot)

![image](https://user-images.githubusercontent.com/22732118/175202449-5069e6a9-c7dd-4824-8957-9ea8a7cf7061.png)

When I try to upload smaller image file size, it continue to upload without any error. But with bigger file size, the error above occurs.

Any help is appreciated.

**Comment by joshhanley** [comment, marked as answer]
@isaacdarcilla have a look at what your global validation is set to, maybe it is blocking it https://laravel-livewire.com/docs/2.x/file-uploads#global-validation

Hope this helps!

**Reply by isaacdarcilla** [reply]
Did not see this but it worked. Thanks!

**Reply by joshhanley** [reply]
No worries! 🙂
